### DISCUSSION TITLE
Locale

### DISCUSSION THREAD

**Discussion by RafeAldridge** [firefly-iii/firefly-iii#4425] [Firefly III]
I am running Firefly III version 5.4.6.

No matter what I try, I can't get this message to disappear: 

"Firefly III is unable to format monetary amounts because your server is missing the required packages."

I am running Windows. The language and formatting is all perfect, I just have this message!

I've tried setting the .env locale to en-GB and en_GB neither of which help :(

Thanks. 

Debug information generated at 2021-02-22 21:23:45 Europe/London for Firefly III version **5.4.6**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.4.6 |
| Firefly III API | 1.4.0 |
| PHP | 7.4.14 |
| Host | WINNT |

| System info | Value |
| --- | --- |
| Installation ID | e7157ead-e0fa-44d8-9360-0a3a7cf9fb28 |
| Using docker? | false |
| Telemetry | false |
| Layout | v1 |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en-GB |
| Default locale | en-GB |
| BCscale | 12 |
| DB drivers | mysql, sqlite |
| Current driver | mysql |
| DB version | 15 (exp. 15) |
| Login provider | eloquent  |
| Trusted proxies (.env) |  |

| User info | Value |
| --- | --- |
| Session start | 2021-02-01 00:00:00 |
| Session end | 2021-02-28 23:59:59 |
| Session first | 2021-01-01 00:00:00 |
| User ID | 2 |
| User language | en_GB |
| User locale | en-GB |
| Attempt at "en-GB.utf8" | false |
| Attempt at "en-GB.UTF-8" | false |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.182 Safari/537.36 |

```
[2021-02-22 21:23:43] local.ERROR: Could not set locale. The following array doesnt work:  ["en-GB.utf8","en-GB.UTF-8"] 

```

**Comment by sephrat** [comment]
Did you take a look at https://github.com/firefly-iii/firefly-iii/issues/4012 ?
I know OP's issue wasn't solved but there are some leads there in regards to Windows locales.
